### PR TITLE
chore(stacks): add new mariadb galera

### PR DESCRIPTION
**Pull request by yanavasileva**
Old image of galera seems to not be working (not sure if it was working before)
We need to test a failure related to the patch release that is scheduled for the end of the week.
I would like to have a local setup as close as possible to CI

### CODE REVIEW

File: stack-templates.json [hunk 1]
```diff
@@ -95,6 +95,21 @@
     "title": "MariaDB v10.3",
     "type": 2
   },
+  {
+    "categories": [
+      "database"
+    ],
+    "description": "MariaDB GALERA NEW",
```
Comment (maxdanilov) [inline stack-templates.json]: maybe we can have a more descriptive name here?
```suggestion
    "description": "MariaDB Galera v25",
```

(same way as for e.g. Postgresql Stacks)

File: stack-templates.json [hunk 2]
```diff
@@ -95,6 +95,21 @@
...
+    "logo": "https://portainer-io-assets.sfo2.digitaloceanspaces.com/logos/mariadb.png",
+    "note": "List: <a href=\"https://github.com/camunda-ci/portainer-templates/blob/master/stack-templates.json\">https://github.com/camunda-ci/portainer-templates/blob/master/stack-templates.json</a><br/>Source: <a href=\"https://github.com/camunda-ci/portainer-templates/blob/master/stacks/camunda-ci-mariadb-galera-new/docker-stack.yml\">https://github.com/camunda-ci/portainer-templates/blob/master/stacks/camunda-ci-mariadb-galera-new/docker-stack.yml</a>",
+    "platform": "linux",
+    "repository": {
+      "stackfile": "stacks/camunda-ci-mariadb-galera-new/docker-stack.yml",
+      "url": "https://github.com/camunda-ci/portainer-templates"
+    },
+    "title": "MariaDB GALERA NEW",
```
Comment (maxdanilov) [inline stack-templates.json]: same here

**Review by maxdanilov — CHANGES_REQUESTED** [review]
According to readme https://github.com/camunda-ci/portainer-templates#how-to-create-a-new-stack-type-template you need to change `new-stack-templates.json` and run `python3 generate-stack-templates.py`

**Comment by cmur2** [comment]
I don't see the change to `new-stack-templates.json` as part of this PR?

**Comment by yanavasileva** [comment]
>change new-stack-templates.json and run python3 generate-stack-templates.py

That will change the stack-template that I already performed, right?
I don't have set python3 configured

**Review by maxdanilov — APPROVED** [review]

**Review by maxdanilov — APPROVED** [review]

**Pull request merged**